Bearing and Azimuth direction support for quantity framework (#7042)
iTwin/itwinjs-core: docs/bis/ec/ec-format.md

- `fractional`
- `scientific`
- `station`

> - `bearing`

Reviewer: we have to be careful here ... this is not supported in EC at this point.  When it is we will need to identify that it's only available in a new EC version ... I guess 3.3

Author: I added a note saying this is only supported in the quantity package at the moment

**thousandSeparator** the character used to separate the integer part of a value in groups of 3 digits. If not defined a localized separator will be used.

> **uomSeparator** the character used to separate the magnitude and the Unit label. For composite formats applies to each component.

Reviewer 2: ```suggestion
**uomSeparator** the character used to separate the magnitude and the Unit label. For composite formats, this applies to each component.
```

**stationSeparator** the character used to separate the station and off set portions of a `station` formatted value.

> **azimuthBase** A numeric value indicating the base when type is set to azimuth. It fet, the `azimuthBaseUnit` has to also be set to indicate which unit this value is in. Defaults to north (quarter rotation). The value is indicated from east counter-clockwise.

Reviewer 2: ```suggestion
**azimuthBase** A numeric value indicating the base when type is set to azimuth. If set, the `azimuthBaseUnit` has to also be set to indicate which unit this value is in. Defaults to north (quarter rotation). The value is indicated from east counter-clockwise.
```
